mathcomp/ssreflect/Makefile.coq-makefile

```diff
@@ -1,22 +1,24 @@
 define coqmakefile
 	(echo "Generating Makefile.coq for Coq $(V) with COQBIN=$(COQBIN)";\
 	if [ "$$OS" = "Windows_NT" ]; then LN=cp; else LN="ln -sf"; fi;\
-	MLLIB=ssreflect_plugin.mlpack;\
+	MLLIB=;\
 	EXTRA=;\
 	case $(V) in\
        	v8.5*|v8.4*)\
 		$$LN $(1)/plugin/$(V)/ssrmatching.mli .;\
 		$$LN $(1)/plugin/$(V)/ssrmatching.ml4 .;\
 		$$LN $(1)/plugin/$(V)/ssrmatching.v .;\
 		$$LN $(1)/plugin/$(V)/ssreflect_plugin.mllib .;\
-		EXTRA="ssrmatching.mli ssrmatching.ml4 ssrmatching.v";\
+		EXTRA="ssrmatching.mli ssrmatching.ml4 ssrmatching.v ssreflect.ml4";\
 		MLLIB=ssreflect_plugin.mllib;\
+		$$LN $(1)/plugin/$(V)/ssreflect.ml4 .;\
 	;;\
-	*)\
+	v8.6*)\
 		$$LN $(1)/plugin/$(V)/ssreflect_plugin.mlpack .;\
+		$$LN $(1)/plugin/$(V)/ssreflect.ml4 .;\
+		MLLIB=ssreflect_plugin.mlpack;\
```

Comment:
It is surprising, but was already like that before my PR, right? Do you confirm it's a bug?